forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt

```diff
@@ -0,0 +1,9 @@
+Name:Morgul-Knife Wound
+ManaCost:1 B
+Types:Enchantment Aura
+K:Enchant creature
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Curse
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddPower$ -3 | AddTrigger$ TrigExile | Description$ Enchanted creature gets -3/-0
+SVar:TrigExile:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ Player.EnchantedController | TriggerZones$ Battlefield | Execute$ TrigExileEnchanted | TriggerDescription$ Enchanted creature gets -3/-0 and has "At the beginning of your upkeep, exile this creature unless you pay 2 life."
```

Comment:
Are you sure that finds the right player?

Reply:
I think so yeah 
![image](https://github.com/Card-Forge/forge/assets/67333662/08993bb4-63be-45b5-a9b9-c2b6862600f9)

Reply:
you're right it trigger on every player's upkeep. Fixed